packages/cozy-realtime/README.md

```diff
@@ -44,6 +44,37 @@ In order to listen to online and offline events, you should install the [Network
 
 ## Example
 
+### With CozyClient
+
+CozyRealtime comes with a plugin for CozyClient. This is the recommended way to use it: 
+```js
+//instanciation
+import { RealtimePlugin } from 'cozy-realtime'
+import { CozyClient} from 'cozy-client'
+
+const client = new CozyClient({})
+
+client.registerPlugin(RealtimePlugin)
+
+//usage
+
+this.realtime = client.plugins.realtime
+// To subscribe
+await this.realtime.subscribe('created', type, handleCreate)
+await this.realtime.subscribe('updated', type, handleCreate)
+await this.realtime.subscribe('updated', type, id, handleUpdate)
+
+// To unsubscribe
+await this.realtime.unsubscribe('created', type, handleCreate)
+await this.realtime.unsubscribe('updated', type, handleCreate)
+await this.realtime.unsubscribe('updated', type, id, handleCreate)
+
+// To unsubscribe all
+await this.realtime.unsubscribeAll()
+
+
+```
```

Comment:
Too much line breaks, and missing line breaks before and after "### Standalone".